src/main.js

```diff
@@ -490,7 +500,10 @@ define('peaks', [
     if (this.player) {
       this.player.destroy();
     }
+    this.emit('destroyed');
   };
 
+  Peaks.CueEmitter = CueEmitter;
```

Comment:
I'd prefer to avoid exporting `CueEmitter`, it's an implementation detail that shouldn't be exposed to users.

Reply:
ah yes, my bad, forgot it there..